### PR TITLE
Add patch to install_name_tool to work on a file copy for osx-arm64

### PR DESCRIPTION
**Pull request by erykoff**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/patches/Modify-install_name_tool-to-work-on-a-file-copy-on-Apple-Silicon.patch [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+ cctools/misc/install_name_tool.c | 6 +++---
+ 1 file changed, 3 insertions(+), 3 deletions(-)
+
+diff --git a/cctools/misc/install_name_tool.c b/cctools/misc/install_name_tool.c
+index a4b4368..cc80ce2 100644
+--- a/cctools/misc/install_name_tool.c
++++ b/cctools/misc/install_name_tool.c
+@@ -26,7 +26,7 @@
+ #include <limits.h>
+ #include <unistd.h>
```
Comment (katietz) [inline recipe/patches/Modify-install_name_tool-to-work-on-a-file-copy-on-Apple-Silicon.patch]: The use of unistd.h unconditional in source will lead to build failures on none unix (or unix-like) architectures.  Isn't there a macro guard like 'HAVE_UNISTD_H'?  Btw why you need to include that header? Isn't stdlib.h not working too?

File: recipe/patches/Modify-install_name_tool-to-work-on-a-file-copy-on-Apple-Silicon.patch [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+ #include "stuff/port.h" /* cctools-port: fake signing */
+@@ -391,7 +391,7 @@ char *input)
+     struct load_command *lc;
+     enum byte_sex host_byte_sex;
+ 
+-#if defined(__APPLE__) && __aarch64__
++#if defined(__APPLE__) && __aarch64__ /* cctools-port: copy before modification */
+         char tempfile[PATH_MAX];
+ 
+ 	snprintf(tempfile, PATH_MAX, "%sXXXXXX", input);
```
Comment (katietz) [inline recipe/patches/Modify-install_name_tool-to-work-on-a-file-copy-on-Apple-Silicon.patch]: I guess you used unistd.h for this prototype.  Actually the proper include here is #include <stdio.h> instead for having this prototype on unix. For windows-sake you should addtionally include stdlib.h ... but in general stdio.h should be fine here too.

**Review by katietz — CHANGES_REQUESTED** [review]

File: recipe/patches/Modify-install_name_tool-to-work-on-a-file-copy-on-Apple-Silicon.patch (hunk 1 above)
Comment (erykoff) [reply recipe/patches/Modify-install_name_tool-to-work-on-a-file-copy-on-Apple-Silicon.patch]: This include is in the original upstream, not in my patch.  And this package can't build on windows in general (per the `meta.yaml`).  I don't want to mess with the upstream includes, which I assume are there for a reason.

**Comment by erykoff** [comment]
So it turns out this patch was just merged upstream in https://github.com/tpoechtrager/cctools-port/pull/98 .  So this is a dead-letter PR.  I'll make a new one that simply updates to the latest version of cctools-port.

**Pull request closed without merging**